Add notice for doesn't support to use cross account ECR image when creating Lambda function (lambda-images.md) (#230)
awsdocs/aws-lambda-developer-guide: doc_source/lambda-images.md

There is no additional charge for packaging and deploying functions as container images\. When a function deployed as a container image is invoked, you pay for invocation requests and execution duration\. You do incur charges related to storing your container images in Amazon ECR\. For more information, see [Amazon ECR pricing](http://aws.amazon.com/ecr/pricing/)\. 

> **Note**: Using the container image currently only support for the image from Amazon ECR in the same AWS account.

Reviewer: ```suggestion
**Note**: Using a container image currently only supports Amazon ECR images in the same AWS account.
```
